video_xblock/static/js/studio-edit.js

```diff
@@ -579,6 +638,14 @@ function StudioEditableXBlock(runtime, element) {
         authenticateVideoApi($data);
     });
 
+    $3playmediaTranscriptsApi.on('click', function(event) {
+        event.preventDefault();
+        event.stopPropagation();
+        var $api_key = $('.threeplaymedia-api-key', element).val();
```

Comment:
In this case yes. We're getting this API key from the user in the first place. And the key does not leave the same view where user enters it.